server/routerlicious/packages/routerlicious/src/deli/index.ts

```diff
@@ -211,6 +212,13 @@ export async function deliCreate(
 		serviceConfiguration,
 		customizations?.clusterDrainingChecker,
 	);
+
+	deliLambdaFactory.on("dispose", async () => {
+		broadcasterLambda.close();
+		await publisher.close();
```

Comment:
Technically, nothing is awaiting this promise, so if `publisher.close` throws, this will throw an uncaught exception. I'd recommend removing the `async` prefix, and adding a `.catch()` to the publisher.close call.